src/components/ItemsList/ItemsList.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+
+export default function ItemsList({ items, deleteItem, editItem }) {
```

Comment:
I encourage you to find a way to have the edit inputs behave individually instead of editing all at once sharing the same input value.